doc/en/multitest.rst

```diff
@@ -1026,18 +1029,23 @@ method has default values assigned to the parametrized arguments:
 Testcase name generation
 ++++++++++++++++++++++++
 
-When you use parametrization, Testplan will try to generate a legible name for
+When you use parametrization, Testplan will try to generate a function name for
```

Comment:
Generated function name is implementation detail user need not to know. Please remove this part.

Reply:
removed